src/utils/prettyPrint/index.ts

```diff
@@ -0,0 +1,89 @@
+import { parse, stringify } from "lossless-json";
+
+/**
+ * `findJsonObjects` finds the indices of json objects in a given log line.
+ * @param logLine - the log line in which to search for json objects
+ * @returns an array of [startIndex, endIndex] representing the position of each json object in the string
+ */
+export const findJsonObjects = (logLine: string) => {
```

Comment:
I want to say it's slightly different because it seems like `getJsonString` can only get one JSON object in a string. However it is possible for there to be multiple JSON objects in a resmoke line, like below:

<img width="921" alt="Screen Shot 2022-10-25 at 1 01 39 PM" src="https://user-images.githubusercontent.com/47064971/197836903-7a4906b3-dbb4-4bb2-9a0d-c45bc9b282c6.png">

Reply:
Do you think it would make sense to only use your function or is there still value in having both? If the latter can we add some documentation to both of these on when to use each one?